src/reason_parser.mly

```diff
@@ -3980,84 +4154,96 @@ constructor_arguments:
       }
   | non_arrowed_simple_core_type non_arrowed_simple_core_type_list
       {
-        let core_type_loc = mklocation $startpos($1) $endpos($1) in
+        let core_type_loc = mklocation $symbolstartpos $endpos($1) in
         let ct = only_core_type $1 core_type_loc in
         Pcstr_tuple (ct :: List.rev $2)
       }
 ;
 
 
 label_declaration:
-    mutable_flag as_loc(LIDENT) attributes
+    attributes mutable_flag as_loc(LIDENT)
       {
        let loc = mklocation $symbolstartpos $endpos in
-       let ct = mkct $2 in
-       (Type.field $2 ct ~mut:$1 ~loc, $3)
+       let ct = mkct $3 in
+       (Type.field $3 ct ~mut:$2 ~loc, $1)
       }
-  |  mutable_flag as_loc(LIDENT) attributes COLON poly_type attributes
+  |  attributes mutable_flag as_loc(LIDENT) COLON poly_type
```

Comment:
This is a potential area for concern, within the AST it's possible to set an attribute here - although it's not clear to me which syntax would trigger this.